scan-command/src/main/java/io/ballerina/scan/utils/RuleDescription.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan.utils;
+
+/**
+ * {@code RuleDescription} provides descriptions for Ballerina static coding rules.
```

Comment:
Still not consistent with others.

E.g.,

Rule's is

```java
{@code Rule} represents a static code analysis rule.
```

We shouldn't then say

```
Ballerina static coding rules.
```